WalletWasabi/BitcoinCore/CoreNode.cs

```diff
@@ -257,9 +257,9 @@ public async Task<bool> TryStopAsync(bool onlyOwned = true)
 		{
 			await DisposeAsync().ConfigureAwait(false);
 
-			Exception exThrown = null;
+			Exception? exThrown = null;
```

Comment:
No. This uses an exception instance as a flag (`null` or `not null`) and we have booleans for things like that. Use a boolean here. I am sure the `bridge` can be also made non-nullable but I would need to see sit and think a bit more.

Reply:
I removed it completely.